backend/src/routes/blog.ts

```diff
@@ -320,4 +321,29 @@ blogRouter.get("/bulkUser/:id", async (c) => {
       message: "Error while fetching post",
     });
   }
+});
+interface ChatCompletionMessageParam {
+  role: 'system' | 'user' | 'assistant';
+  content: string;
+  name?: string;
+}
```

Comment:
Is this used anywhere in blog.ts ?

Reply:
Oh no. my bad for that. will be creating a commit with the changes soon.